my-app/src/App.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+//import logo from './logo.svg';
+import './App.css'
+import { Card } from './components/Card/box' //card
+import { PopupCard } from './components/Card/popupCard' //card
+import { Form } from './components/form'
+import { Footer } from './components/footer'
+//import { Connection } from './components/api';
+//import { CardList } from './components/cardList';
+import Carousel from 'react-elastic-carousel'
+
+const accesKey = 'rQ-_SmO3ayd_uvDdmk1atdqJifxUQahY2IdZM90ux6k'
+const endPoint = 'https://api.unsplash.com/search/photos'
+const breakpoint = [
+  { width: 500, itemsToShow: 1 },
+  { width: 720, itemsToShow: 2 },
+  { width: 1200, itemsToShow: 3 },
+  { width: 1500, itemsToShow: 4 },
+]
+
+export class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      error: null,
+      isLoaded: false,
+      abierto: false,
+      id: 1,
+      items: [],
+    }
+  }
+
+  componentDidMount() {
+    fetch(endPoint + '?query=new-york&client_id=' + accesKey)
```

Comment:
Since you're moving all API calls to axios, I suggest having all of them in a separated file, so you don't have to rewrite all the endpoint everytime you're using it.